src/targets/gpu/fuse_ops.cpp

```diff
@@ -903,10 +971,15 @@ void fuse_ops::apply(module& m) const
     match::find_matches(m, find_conv_pointwise{ctx}, find_conv_bias_relu{ctx}, find_conv_bias{ctx});
     run_passes(m, {dead_code_elimination{}});
 #endif
-    match::find_matches(m,
 #if MIGRAPHX_USE_ROCBLAS
-                        find_gemm_pointwise{},
+    if(not enabled(MIGRAPHX_ENABLE_HIPBLASLT_GEMM{}) or not hipblaslt_supported())
```

Comment:
You dont need to check this at runtime since it was already checked in lowering.

Reply:
This makes the test case fail (when hipblaslt is not enabled) because the test case is expecting `hipblaslt_op` in:
`@4 = gpu::hipblaslt_op[op=gpu::hip_gemm[alpha=1,beta=1,solution_idx=0]](a,b,x,@0) -> float_type, {1, 3, 3}, {9, 3, 1}`
but it gets:
`@4 = gpu::gemm[alpha=1,beta=1,compute_fp32=1,trans_batch=0,solution_idx=0](a,b,x,@0) -> float_type, {1, 3, 3}, {9, 3, 1}`.

Reply:
I could modify the test case so it works for rocblas as well, and then we can remove this check.

Reply:
I have modified the test case so it works for both rocblas and hipblaslt.

Reply:
Removed the check.